includes/acl/compression/impl/normalize_streams.h

```diff
@@ -100,28 +100,114 @@ namespace acl
 		{
 			const rtm::vector4f one = rtm::vector_set(1.0F);
 			const rtm::vector4f zero = rtm::vector_zero();
+
+#ifdef ACL_PRECISION_BOOST
+
+			const float max_range_value_flt = float((1 << k_segment_range_reduction_num_bits_per_component) - 2);
+
+#else
+
 			const float max_range_value_flt = float((1 << k_segment_range_reduction_num_bits_per_component) - 1);
+
+#endif
+
 			const rtm::vector4f max_range_value = rtm::vector_set(max_range_value_flt);
 			const rtm::vector4f inv_max_range_value = rtm::vector_set(1.0F / max_range_value_flt);
 
+#ifdef ACL_PRECISION_BOOST
+
+			const rtm::vector4f half = rtm::vector_set(0.5F);
+			const rtm::vector4f half_neg = rtm::vector_set(-0.5F);
+			const rtm::vector4f half_range_value = rtm::vector_set(float((1 << (k_segment_range_reduction_num_bits_per_component - 1)) - 1));
+			// Segment ranges are always normalized and live between [-0.5 ... 0.5]
+
+#else
+
 			// Segment ranges are always normalized and live between [0.0 ... 1.0]
 
+#endif
+
 			auto fixup_range = [&](const track_stream_range& range)
 			{
+
+
+#ifdef ACL_PRECISION_BOOST
+
+				// In our compressed format, we store the center value of the track range quantized on 8 bits.
+				// To get the best accuracy, we pick the values closest to the true minimum that is slightly lower, and true maximum that is slightly higher.
+				// This is to ensure that we encompass the lowest and highest values even after quantization.
+				const rtm::vector4f range_min = range.get_min();
+				const rtm::vector4f quantized_min0 = rtm::vector_clamp(rtm::vector_add(rtm::vector_floor(rtm::vector_mul_add(range_min, max_range_value, half)), half_range_value), zero, max_range_value);
```

Comment:
Note: range_min is already normalized in the [-0.5, 0.5] range here, we convert it back into [0.0, 1.0] range after symmetric rounding